oct/cli/package/common_options.py

```diff
@@ -13,22 +13,44 @@ def package_options(func):
     :param func: Click CLI command to decorate
     :return: decorated CLI command
     """
+    option(
+        '--stage',
+        '-s',
+        'upgrade_stage',
+        type=Choice(['current', 'next', 'fork', 'base']),
```

Comment:
Just to make sure we're on the same page, I see the tree as:

```
stages/
└── bare
    ├── base
    │   └── build
    │       └── install
    └── fork
```

So you want to let a user go forward, stay the same, and then explicitly give the choice for the only place it's nonlinear. OK. I mis-understood this as giving the user the choice to do any AMI stage. I like your approach.